build/dependencies.yaml

```diff
@@ -256,7 +256,7 @@ dependencies:
     - path: test/e2e_node/resource_collector.go
       match: gcr.io\/cadvisor\/cadvisor:v\d+\.\d+\.\d+
     - path: test/e2e_node/image_list.go
-      mathc: gcr.io\/cadvisor\/cadvisor:v\d+\.\d+\.\d+
+      match: gcr.io\/cadvisor\/cadvisor:v\d+\.\d+\.\d+
```

Comment:
I wonder what the effect of this mistake was - apparently none or someone would have noticed? Do we we need to fix something else because of it?

cc @kubernetes/sig-release-leads

Reply:
Apparently, zeitgeist is just treating that entry as a noop. Correcting it should not have any effect as the pinned version matches what's in the source code:

https://github.com/kubernetes/kubernetes/blob/b98b86bd6abb24e3d2451a885354f09cd7af22a2/test/e2e_node/image_list.go#L54

Reply:
Would it be worth changing zeitgeist to fail on unknown fields ("strict decoding")? That would have caught the mistake.